### PR TITLE
Install analytics plugin by default on strapi new project

### PR DESCRIPTION
**Pull request by lauriejim**
💅 Enhancement

- Handle no core plugins installation on strapi new

### CODE REVIEW

File: packages/strapi-generate-new/lib/after.js [hunk 1]
```diff
@@ -83,15 +83,33 @@ module.exports = (scope, cb) => {
...
+      name: 'analytics',
+      core: false
+    }];
 
     // Install each plugin.
     defaultPlugins.forEach(defaultPlugin => {
       try {
-        execSync(`node ${strapiBin} install ${defaultPlugin} ${scope.developerMode ? '--dev' : ''}`);
-        logger.info(`The plugin ${defaultPlugin} has been successfully installed.`);
+        execSync(`node ${strapiBin} install ${defaultPlugin.name} ${scope.developerMode && defaultPlugin.core ? '--dev' : ''}`);
```
Comment (Aurelsicoko) [inline packages/strapi-generate-new/lib/after.js]: `defaultPlugin.core` >>> `defaultPlugins.core` (s is missing)

Comment (Aurelsicoko) [reply packages/strapi-generate-new/lib/after.js]: Also for the `.name`

Comment (lauriejim) [reply packages/strapi-generate-new/lib/after.js]: `defaultPlugins` is an array and `defaultPlugin` the object.

Comment (Aurelsicoko) [reply packages/strapi-generate-new/lib/after.js]: Oups, I didn't see the `forEach` loop

**Pull request merged**